cmd/create/execution.go

```diff
@@ -0,0 +1,123 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/lyft/flytectl/cmd/config"
+	cmdCore "github.com/lyft/flytectl/cmd/core"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/core"
+
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+)
+
+const (
+	executionShort = "Create execution resources"
+	executionLong  = `
+Create the executions for given launchplan in a project and domain.
+Currently the launchplan and where the execution needs to be launched , need to belong to same domain and project.
+
+There are three steps in generating an execution.
+
+- Generate the execution param yaml file containing in the default input parameters for the launch plan.
+- If the default parameters exist then you can modify the default param values in the generated yaml file.
+- Run the execution passing in the generated yaml file.
+
+The following command would generate core.basic.lp.go_greet.inputs.yaml file by using the launch plan provided in argument.
+This would use the latest version of the launch plan.
+::
+
+ bin/flytectl create execution core.basic.lp.go_greet  -d development  -p flytesnacks -g
+
+The generated file would look similar to this
+
+.. code-block:: yaml
+
+	inputs:
+	  am: false
+	  day_of_week: ""
+	  number: 0
+
+The generated file can be modified to change the parameter values and can be passed to command for execution
+::
+
+ bin/flytectl create execution core.basic.lp.go_greet -f core.basic.lp.go_greet.inputs.yaml  -d development  -p flytesnacks
+
+Usage
+`
+)
+
+//go:generate pflags ExecutionConfig --default-var executionConfig
+
+// ProjectConfig Config hold configuration for project create flags.
+type ExecutionConfig struct {
+	File     string `json:"file" pflag:",file for the execution params.If not specified defaults to <launchplan_name>.inputs.yaml"`
+	Version  string `json:"version" pflag:",version of the launch plan to be executed."`
+	GenParam bool   `json:"genParam" pflag:",bool flag to indicate the generation of params file."`
```

Comment:
```suggestion
	GenParam bool   `json:"generate-params-file" pflag:",bool flag to indicate the generation of params file."`
```

Reply:
Also can this be a string argument, where the user can specify the path/name of the file to be created?

Reply:
So here was my thought,
generateParamsFile is the boolean flag which indicates whether its a 
1] params file generation mode (write) if its true
    and writes to path specified using the File option.
    If the  File option is not specified then it defaults to writing it to <launchplan_name>.inputs.yaml
or 
2] read mode if its false where the path in the File option is used to read the params from the yaml file.
    If the File option is not specified then similar to read case we will try to read <launchplan_name>.inputs.yaml

Let me know what you think

Reply:
that makes sense